soccer/planning/Path.hpp

```diff
@@ -15,8 +15,11 @@ namespace Planning {
  */
 class Path {
 public:
+    Path() : _startTime(0) {}
     virtual ~Path() {}
 
+    virtual bool valid() const = 0;
```

Comment:
Good point, I think checking that `destination` is non-null is just as good.  Just removed it.